sparse_ho/models/svr.py

```diff
@@ -392,6 +389,8 @@ def generalized_supp(self, X, v, log_hyperparam):
         return v[full_supp]
 
     def proj_hyperparam(self, X, y, log_hyperparam):
+        # XXX this function operates inplace while returning the
+        # update value. It's confusing
         if log_hyperparam[0] < -16.0:
```

Comment:
@Klopfe not really related (apart from cleanup), but I saw this scheme a few times in the code and you can use np.clip to clip a value in a given range (using 1 line instead of 4)

Reply:
yes thanks! I will use this function from now on